index.html

```diff
@@ -65,7 +65,7 @@ <h2 class="footnote">
         </dt>
         <dd>Final batch of acceptance notifications</dd>
         <dt>
-          October 5, 2016:
+          October 7, 2016:
```

Comment:
should probably update the date on the [Sponsorship Packages page](https://github.com/w3c/vr-workshop/blob/4a3efbe/sponsorship.html#L60) too